olp-cpp-sdk-core/include/olp/core/client/ApiResponse.h

```diff
@@ -60,7 +145,24 @@ class ApiResponse {
    * @param result The `ResultType` instance.
    */
   ApiResponse(ResultType result)  // NOLINT
```

Comment:
Why not set to `explicit` instead of // NOLINT.

Reply:
This is not backward compatible with existing code.